packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -437,7 +437,7 @@ const AccountSwitcher = props => {
                                             <AccountList
                                                 is_dark_mode_on={props.is_dark_mode_on}
                                                 key={account.login}
-                                                market_type={account.market_type}
+                                                account_type={account.market_type}
```

Comment:
why are we changing the prop name? 
`account_type` is different from `market_type`, it will be confusing if we switch the names.

```
account_type: "real" | "demo"
market_type: "financial" | "gaming"
```

Reply:
Yeah that was a mistake, I reverted the name change and made it market_type again.